demos/graph-classification/dgcnn-graph-classification.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>Is this cell still relevant now that there's the <code>predictions.shape</code>  cell above, and now that we've got the Flatten? It seems a bit peculiar to run predictions on an untrained model and so could maybe ether be deleted or have some explanation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>It's not, I forgot to delete it. In fact, we don't need the other two cells above printing the shapes for predictions and x_out. I was trying to debug the model and forgot to delete them.</p>